proto/viam/service/mlmodels/v1/mlmodels.proto

```diff
@@ -0,0 +1,92 @@
+syntax = "proto3";
+
+package viam.service.mlmodels.v1;
```

Comment:
why not mlmodel

Reply:
mostly because we have another service called "sensors" rather than "sensor". But then, I guess it's called the "SensorsService". I like MLModel as the service name more, and will change it

Reply:
changed to mlmodel